src/app.jsx

```diff
@@ -15,8 +15,8 @@ class App extends React.Component {
       avgReviews: 0,
       totalReviews: 0
     };
-    this.informationip = env.INFORMATION_IP || localhost;
-    this.reviewsip = env.REVIEW_IP || localhost;
+    this.informationip = /* env.INFORMATION_IP || */ 'localhost';
```

Comment:
what does this line do?

Reply:
This service pulls from other services. I decided to comment out the external service request temporarily. I found that the env object wasn't working as intended, so I just set it to localhost for testing purposes.